lib/theme.dart

```diff
@@ -141,15 +141,17 @@ class AppTheme {
     );
   }
 
-  static ThemeData getThemeFromPreferences(bool automatic, bool dark) {
-    DateTime now = DateTime.now();
-    if (automatic) {
-      if (now.hour >= 20 || now.hour < 8) return AppTheme.darkTheme();
-      else return AppTheme.lightTheme();
-    } else {
-      if (dark) return AppTheme.darkTheme();
+  static ThemeData getNormalThemeFromPreferences(bool isAutomatic, bool isDark) {
+    if (isAutomatic) return AppTheme.lightTheme();
+    else {
+      if (isDark) return AppTheme.darkTheme();
```

Comment:
For simple ternary operation use `return isDark ? AppTheme.darkTheme() : AppTheme.LightTheme();`. But it's the same so don't fix this line.